src/style.css

```diff
@@ -0,0 +1,390 @@
+@font-face {
+  font-family: "DINProRegular";
+  src: url("/fonts/DINProRegular.otf") format("opentype");
+}
+
+body {
+  margin: 10px 0px 0px 10px;
+  font-size: 13px;
+}
+
+form {
+  background-color: white;
+  font-family: 'DINProRegular';
+  width: 275px;
+  box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.01), 0 3px 24px rgba(48, 51, 51, 0.09);
+  border-radius: 2px;
+}
+
+header {
+  padding-right: 24px;
+  padding-left: 24px;
+  display: flex;
+  border-bottom: 1px solid #f5f5f6;
+  height: 52px;
+  align-items: flex-end;
+}
+
+header div button {
+  font-family: 'DINProRegular';
```

Comment:
I would consider adding some fall-back fonts (see font-family) for more robust typography. In this case, the font is loaded by webpack and served from the client. However, this could be relevant when using google fonts, as the font may take time to load or the client's browser may have firewalls preventing it from accessing the font. In any case, if the otf file is corrupted or some special characters are missing, it would be ideal to specify a sans serif font. Without fallback fonts, the defaults could mess with your spacing.